sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/cbs.py

```diff
@@ -169,7 +204,7 @@ def _cbs_link_ready(self):
         if self.state != CbsState.OPEN:
             return False
         if self.state in (CbsState.CLOSED, CbsState.ERROR):
-            raise TokenAuthFailure(
+            raise TokenAuthFailure( # pylint: disable = no-value-for-parameter
```

Comment:
@kashifkhan - this is expecting `status_code` and `status_description` as parameters. Should these be passed in?

Reply:
`condition` should become `status_code` and `description` => 'status_description`